### ISSUE TITLE
frames are randomly ordered when using wgif on ubuntu

### ISSUE THREAD

**Issue by aparrish** [ecmendenhall/wgif#19]
I installed wgif on ubuntu today and the gifs it made had the frames in essentially random order. I tracked the problem down to [this line](https://github.com/ecmendenhall/wgif/blob/master/lib/wgif/video.rb#L45)—[according to this](http://stackoverflow.com/questions/6220753/does-dir-glob-guarantee-the-order), `.glob` isn't guaranteed to return its results in any particular order. If I put a little `puts frames_dir.inspect` in the `make_gif` method, I get output like this:

```
["/tmp/wgif/frames/00042.png", "/tmp/wgif/frames/00013.png", "/tmp/wgif/frames/00002.png", "/tmp/wgif/frames/00004.png", "/tmp/wgif/frames/00003.png", "/tmp/wgif/frames/00028.png", "/tmp/wgif/frames/00051.png", "/tmp/wgif/frames/00049.png", "/tmp/wgif/frames/00050.png", "/tmp/wgif/frames/00024.png", "/tmp/wgif/frames/00005.png", "/tmp/wgif/frames/00052.png", "/tmp/wgif/frames/00041.png", "/tmp/wgif/frames/00045.png", "/tmp/wgif/frames/00038.png", "/tmp/wgif/frames/00034.png", "/tmp/wgif/frames/00033.png", "/tmp/wgif/frames/00044.png", "/tmp/wgif/frames/00022.png", "/tmp/wgif/frames/00011.png", "/tmp/wgif/frames/00026.png", "/tmp/wgif/frames/00040.png", "/tmp/wgif/frames/00017.png", "/tmp/wgif/frames/00036.png", "/tmp/wgif/frames/00007.png", "/tmp/wgif/frames/00025.png", "/tmp/wgif/frames/00039.png", "/tmp/wgif/frames/00014.png", "/tmp/wgif/frames/00029.png", "/tmp/wgif/frames/00043.png", "/tmp/wgif/frames/00019.png", "/tmp/wgif/frames/00030.png", "/tmp/wgif/frames/00027.png", "/tmp/wgif/frames/00053.png", "/tmp/wgif/frames/00035.png", "/tmp/wgif/frames/00001.png", "/tmp/wgif/frames/00006.png", "/tmp/wgif/frames/00010.png", "/tmp/wgif/frames/00021.png", "/tmp/wgif/frames/00012.png", "/tmp/wgif/frames/00016.png", "/tmp/wgif/frames/00031.png", "/tmp/wgif/frames/00020.png", "/tmp/wgif/frames/00023.png", "/tmp/wgif/frames/00046.png", "/tmp/wgif/frames/00047.png", "/tmp/wgif/frames/00008.png", "/tmp/wgif/frames/00032.png", "/tmp/wgif/frames/00015.png", "/tmp/wgif/frames/00009.png", "/tmp/wgif/frames/00018.png", "/tmp/wgif/frames/00048.png", "/tmp/wgif/frames/00037.png"]
```

(When I run this on OSX, I get the same frames back—but sorted lexicographically.)

I think the fix is pretty easy—just put a `.sort` on the end of [this line](https://github.com/ecmendenhall/wgif/blob/master/lib/wgif/video.rb#L45). (I made this change in my local copy and it works fine—sorry for no pull request, I don't actually know how to write ruby and I'm not quite sure how I'd write unit/regression tests for this.)

**Comment by ecmendenhall** [comment]
Oh my glob, sorry this easy fix was open for so long! Notifications for this project keep getting lost and I just noticed your issue tonight. :unamused: 

You're totally right: `Dir#glob` sorts alphabetically on OS X, but not on Linux/Windows. I added your fix and pushed up [version 0.5.3](https://rubygems.org/gems/wgif/versions/0.5.3) to Rubygems. Looks good on my Ubuntu VM, but I'll wait until you confirm to close the issue.
